bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoFragment.kt

```diff
@@ -370,9 +420,12 @@ private fun NavigationActionBack(
 private fun YourInvoiceScanSection(
```

Comment:
[Request] The function name is not good. Please us sth like InvoicePreviewSection or like this.